.env.test

```diff
@@ -16,3 +16,6 @@ BASE_URL=https://api.trello.com/1/
 
 # Tests
 FLASK_SKIP_DOTENV=True
+
+# sample webhook
+AZURE_WEBHOOK_URL="https://\$<deployment_username>:<deployment_password>@<webapp_name>.scm.azurewebsites.net/docker/hook"
```

Comment:
This isn't necessary in our `.env*` files since it's not used directly by our code so I'd lean towards removing this